packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,238 @@
+# Nimble Combobox (IxD)
+
+## Overview
+
+The combobox component is used to present a pre-defined list of options that end-users can choose from or type to create a custom value. Comboboxes combine the flexibility of text input with the efficiency of selection for more user-friendly interactions.
+
+> THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+
+-   [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+-   [ViD Combobox Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1529-41777&mode=design)
+
+## Usage
+
+When to use:
+
+-   To provide an input for text entry coupled with a popup for quick selection and filtering
+-   To allow end-users to enter custom values
+-   For entering settings or preference values that don't (yet) exist
+-   For single selection
+
+When not to use:
+
+-   When text entry is not needed for selection
+-   When there are fewer than three options
```

Comment:
What would be the recommendation for when there are fewer than 3 options? Particularly because of how helpful the autocomplete is (i.e. the dropdown options) and the necessity to enter any value, I would think you'd still want to use a combobox regardless of how many options you have.

Reply:
With fewer than three options, a radio group (or radio button group, which we don't really have a concept for) should be used. This way the user can see all the options immediately, without requiring a click for progressive disclosure. I've added this info to the "Related Components" section.